external-import/c2iocs/src/requirements.txt

```
@@ -0,0 +1 @@
+pycti==5.8.2
```

The last version could be used: 
```suggestion
pycti==5.9.6
```